.vsts.pipelines/builds/matrix.yml

```diff
@@ -76,3 +76,35 @@ stages:
       setupWindows: true
       skipSmokeTest: true
       failOnPrebuiltBaselineError: false
+
+# RHEL builds only internal
+- ${{ if not(and(eq(variables['System.TeamProject'], 'public'), in(variables['Build.Reason'], 'PullRequest'))) }}:
+  - template: ../stages/stage-jobs-matrix.yml
+    parameters:
+      jobTemplateName: ../jobs/ci-local.yml
+      name: rhel7_x64
+      jobParameters:
+        pool:
+          name: DotNet-SourceBuild-Internal
+          demands:
+          - Agent.OSArchitecture -equals X64
+          - rhel -equals 7
+        scriptPrefix: ./
+        scriptSuffix: .sh
+      matrix:
+        Offline: { type: Offline }
+  - template: ../stages/stage-jobs-matrix.yml
+    parameters:
+      jobTemplateName: ../jobs/ci-local.yml
+      name: rhel8_x64
+      jobParameters:
+        pool:
+          name: DotNet-SourceBuild-Internal
+          demands:
+          - Agent.OSArchitecture -equals X64
+          - rhel -equals 8
+        scriptPrefix: ./
+        scriptSuffix: .sh
+        useSystemLibunwind: true
```

Comment:
RHEL 7 has a `libunwind` package, but RHEL 8 doesn't. This conditional seems reversed?

Reply:
Thanks, flipped this around.